### ISSUE TITLE
Tombstones

### ISSUE THREAD

**Issue by martinthomson** [martinthomson/no-expiry#5]
@sftcd suggests:

> I'd suggest adding a recommendation that it'd be useful to publish a tombstone version for drafts that are no longer under consideration (at that point in time) as I do think we have a small problem with drafts that are going nowhere, but that not being visible to readers.

**Comment by paulehoffman** [comment]
I disagree with the recommendation. Drafts often exist for other reasons, such as to document something for an IANA registry, or to say "I had this idea on this date, but am not pursuing it, but you might". Those should not be overtaken by a tombstone.
I'm not against authors publishing tombstones for ideas they thought were good, but then considered bad. I have done that myself. But that's quite different than "no longer under consideration".

**Comment by sftcd** [comment]
Actually I consider adding e.g. "I had this idea on this date, but am not pursuing it, but you might" to a draft or abstract also makes a fine tombstone draft - IMO a tombstone draft doesn't have to be only that text. So maybe if doing something here, define the term to include both or just describe stuff without that term.

**Comment by paulehoffman** [comment]
I like the idea of our draft adding a general definition of "tombstone draft", suggesting when it can be used, but not suggesting it for when an idea is simply abandoned.

**Comment by sftcd** [comment]
think we might disagree a bit about what exactly to say there, but it's something that  can be sorted out later
